test/logger.spec.ts

```diff
@@ -77,6 +77,23 @@ describe("simple-json-logger", () => {
       // then
       expect(instantiate).not.toThrow();
     });
+
+    it("Should set level when provied in options", () => {
```

Comment:
provided*

Reply:
I think it would be good if you test the another 2 test cases:
 - When receive the log level from process.env;
 - When set the default log level when don't have any other.

Reply:
Done!